src/Generator.Tests/AST/TestAST.cs

```diff
@@ -281,5 +281,12 @@ public void TestMacroLineNumber()
         {
             Assert.AreEqual(98, AstContext.FindClass("HasAmbiguousFunctions").First().Specifiers.Last().LineNumberStart);
         }
+
+        [Test]
+        public void TestImplicitDeclaration()
+        {
+            Assert.AreEqual(false, AstContext.FindFunction("testExplicitFunc").Single().IsImplicit);
```

Comment:
Please just ignore IsExplicit. It works correctly at the moment and does not need any related changes.